docs/pages/release_notes.md

```diff
@@ -18,9 +18,28 @@ This is a {{ site.pmd.release_type }} release.
 
 *   The new Java rule {% rule "java/bestpractices/UseStandardCharsets" %} finds usages of `Charset.forName`,
     where `StandardCharsets` can be used instead.
-    
+
+    This rule is also part of the Quickstart Ruleset (`rulesets/java/quickstart.xml`) for Java.
+
+*   The new Java rule {% rule "java/codestyle/UnnecessaryImport" %} merges the rules
+    {% rule "java/bestpractices/UnusedImports" %}, {% rule "java/codestyle/DuplicateImports" %},
+    {% rule "java/errorprone/ImportFromSamePackage" %}, and {% rule "java/codestyle/DontImportJavaLang" %}.
+
     This rule is also part of the Quickstart Ruleset (`rulesets/java/quickstart.xml`) for Java.
 
+#### Deprecated rules
+
+*   java-bestpractices
+    *   {% rule java/bestpractices/UnusedImports %}: use the rule {% rule java/bestpractices/UnusedImports %} instead
```

Comment:
I think, these now need to point to UnnecessaryImport.. I'll fix this

Reply:
Ah you did this already: a6a3db43b0
Thanks!

Reply:
Thanks for merging to pmd/7.0.x :smile: